tests/FakeItEasy.Specs/ThrowsAsyncSpecs.cs

```diff
@@ -0,0 +1,310 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Threading.Tasks;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+
+    public class ThrowsAsyncSpecs
+    {
+        public interface IFoo
+        {
+            Task BarAsync();
+
+            Task BarAsync(int a);
+
+            Task BarAsync(int a, string b);
+
+            Task BarAsync(int a, string b, bool c);
+
+            Task BarAsync(int a, string b, bool c, double d);
+
+            Task<int> BazAsync();
+
+            Task<int> BazAsync(int a);
+
+            Task<int> BazAsync(int a, string b);
+
+            Task<int> BazAsync(int a, string b, bool c);
+
+            Task<int> BazAsync(int a, string b, bool c, double d);
+        }
+
+        [Scenario]
+        public void NoResultPassExceptionInstance(IFoo fake, Task task)
+        {
+            "Given a fake"
+                .x(() => fake = A.Fake<IFoo>());
+
+            "And an async method of the fake configured to throw asynchronously"
+                .x(() => A.CallTo(() => fake.BarAsync()).ThrowsAsync(new MyException()));
+
+            "When that method is called"

```

Comment:
Oh! I just realized why the "I" is an option at all. The (in our case implicit) 0th part of the BDD script:

```cucumber
As a developer using FakeItEasy in my tests
Given a fake
And an async method of the fake configured to throw asynchronously
When I call that method
```

Or something. But I'm still fine with what you have!